tdp_server/conftest.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+from typing import Dict, Tuple
+
+import pytest
+import yaml
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from tdp.core.collection import Collection
+from tdp.core.collections import Collections
+from tdp.core.dag import Dag
+from tdp.core.service_manager import ServiceManager
+
+from tdp_server.api.dependencies import get_dag, get_service_managers
+from tdp_server.api.openid_dependencies import mock_validate_token, validate_token
+from tdp_server.core.config import settings
+from tdp_server.db.base import Base
+from tdp_server.main import create_app
+
+MOCK_SERVICE_DAG = [
+    {"name": "mock_node_install", "depends_on": []},
+    {"name": "mock_node_config", "depends_on": ["mock_node_install"]},
+    {"name": "mock_node_start", "depends_on": ["mock_node_config"]},
+    {"name": "mock_node_init", "depends_on": ["mock_node_start"]},
+    {"name": "mock_install", "noop": True, "depends_on": ["mock_node_install"]},
+    {
+        "name": "mock_config",
+        "noop": True,
+        "depends_on": ["mock_install", "mock_node_config"],
+    },
+    {
+        "name": "mock_start",
+        "noop": True,
+        "depends_on": ["mock_config", "mock_node_start"],
+    },
+    {"name": "mock_init", "noop": True, "depends_on": ["mock_start", "mock_node_init"]},
+]
+
+
+@pytest.fixture
+def app(mock_runtime):
+    app = create_app()
+    # Disable token validation for tests
+
+    dag, service_managers = mock_runtime
+    app.dependency_overrides[validate_token] = mock_validate_token
```

Comment:
The comment should be just before the code. The line with `dag, service_managers = mock_runtime` is not used to disable token validation.
```suggestion

    dag, service_managers = mock_runtime
    # Disable token validation for tests
    app.dependency_overrides[validate_token] = mock_validate_token
```